server/utils/utils.ts

```diff
@@ -139,7 +139,11 @@ export const formatAssessmentData = (
         subData,
       } as AssessmentArea
     })
-    .sort((a, b) => 0 - (a.criminogenicNeedsScore > b.criminogenicNeedsScore ? 1 : -1))
+    .sort((a, b) =>
+      Number(b.criminogenicNeedsScore) - b.thresholdValue - (Number(a.criminogenicNeedsScore) - a.thresholdValue) > 0
```

Comment:
Can you add tests to make sure it's sorting things in the right order and scoring area?

Reply:
The changes made to the existing tests in this PR should be doing that I think